dfaker/wizard.py

```diff
@@ -1,85 +1,117 @@
+                    }
                 elif pump_name == 'OmniPod':
-                    wizard_reading["bgTarget"] = { "high": access_settings["bgTarget"][0]["high"],
-                                                    "target": access_settings["bgTarget"][0]["target"]}
+                    wizard_reading["bgTarget"] = {
+                        "high": access_settings["bgTarget"][0]["high"],
+                        "target": access_settings["bgTarget"][0]["target"]
+                    }
             elif pump_name == 'Tandem':
-                wizard_reading["bgTarget"] = {"target": access_settings["bgTargets"]["standard"][0]["target"]}
+                target = access_settings["bgTargets"]["standard"][0]["target"]
+                wizard_reading["bgTarget"] = {"target": target}
                 carb_ratio_sched = access_settings["carbRatios"]["standard"]
-                sensitivity_sched = access_settings["insulinSensitivities"]["standard"]
-            
-            sensitivity = tools.get_rate_from_settings(sensitivity_sched, wizard_reading["deviceTime"], "insulinSensitivity")
-            carb_ratio = tools.get_rate_from_settings(carb_ratio_sched, wizard_reading["deviceTime"], "carbRatio")
+                sensitivity_sched = \
+                    access_settings["insulinSensitivities"]["standard"]
+
+            sensitivity = tools.get_rate_from_settings(
+                              sensitivity_sched, wizard_reading["deviceTime"],
+                              "insulinSensitivity")
+            carb_ratio = tools.get_rate_from_settings(
+                             carb_ratio_sched, wizard_reading["deviceTime"],
+                             "carbRatio")
             wizard_reading["insulinSensitivity"] = sensitivity
-            wizard_reading["insulinCarbRatio"] = carb_ratio    
+            wizard_reading["insulinCarbRatio"] = carb_ratio
 
             wizard_reading["recommended"] = {}
-            wizard_reading["recommended"]["carb"] = tools.round_to(wizard_reading["carbInput"] / wizard_reading["insulinCarbRatio"])
-            wizard_reading["recommended"]["correction"] =  0
-            wizard_reading["recommended"]["net"] = (wizard_reading["recommended"]["carb"] + wizard_reading["recommended"]["correction"]
-                                                    - tools.round_to(wizard_reading["insulinOnBoard"]))
+            carb = (wizard_reading["carbInput"] /
+                    wizard_reading["insulinCarbRatio"])
+            wizard_reading["recommended"]["carb"] = tools.round_to(carb)
+            wizard_reading["recommended"]["correction"] = 0
+            corrected_carb = (wizard_reading["recommended"]["carb"] +
+                              wizard_reading["recommended"]["correction"])
+            iob = tools.round_to(wizard_reading["insulinOnBoard"])
+            wizard_reading["recommended"]["net"] = (corrected_carb - iob)
+            wizard_reading["units"] = "mmol/L"
```

Comment:
This is the only real change. Everything else should be refactoring for linting and clarity.